packages/cfd/src/Assets/svgs/trading-platform/ic-appstore-cfds.svg

```diff
@@ -1 +1 @@
-1.688-.096-2.64-.04-.952-.1-1.912-.18-2.88a517.564 517.564 0 01-1.008 2.34 108.944 108.944 0 01-.9 2.124H8.906c-.112-.264-.248-.58-.408-.948-.16-.376-.328-.768-.504-1.176-.168-.408-.34-.816-.516-1.224l-.48-1.116c-.08.968-.14 1.928-.18 2.88-.04.952-.072 1.832-.096 2.64H4.898c.032-.656.068-1.356.108-2.1l.144-2.22c.056-.744.116-1.456.18-2.136.072-.688.148-1.308.228-1.86h1.788zM21.358 9.684v1.596H18.85V18h-1.872v-6.72H14.47V9.684h6.888zM25.322 15.552a1.18 1.18 0 00-.156-.612c-.104-.184-.284-.336-.54-.456-.256-.12-.6-.212-1.032-.276a11.46 11.46 0 00-1.632-.096c.088-.744.16-1.5.216-2.268.056-.768.1-1.488.132-2.16h4.5v1.5h-3c-.016.288-.036.568-.06.84-.016.264-.036.496-.06.696 1.168.08 2.028.356 2.58.828.56.464.84 1.112.84 1.944 0 .384-.068.74-.204 1.068-.136.328-.34.612-.612.852a2.9 2.9 0 01-1.032.564c-.408.136-.888.204-1.44.204-.216 0-.444-.016-.684-.048a6.41 6.41 0 01-.684-.108 7.294 7.294 0 01-.588-.132 2.309 2.309 0 01-.408-.144l.324-1.476c.2.088.464.176.792.264.328.08.72.12 1.176.12.552 0 .952-.108 1.2-.324.248-.224.372-.484.372-.78zM11.014 29.669c-.676 0-1.192-.188-1.548-.564-.352-.376-.528-.91-.528-1.602 0-.344.054-.65.162-.918.108-.272.256-.5.444-.684.188-.188.412-.33.672-.426.26-.096.542-.144.846-.144.176 0 .336.014.48.042.144.024.27.054.378.09a1.678 1.678 0 01.426.186l-.27.756a2.248 2.248 0 00-.45-.174 2.1 2.1 0 00-.576-.072c-.144 0-.286.024-.426.072a.97.97 0 00-.366.234c-.104.104-.188.24-.252.408a1.723 1.723 0 00-.096.612c0 .192.02.372.06.54.044.164.112.306.204.426.096.12.22.216.372.288.152.068.336.102.552.102.136 0 .258-.008.366-.024.108-.016.204-.034.288-.054a1.77 1.77 0 00.222-.078l.174-.078.258.75c-.132.08-.318.152-.558.216-.24.064-.518.096-.834.096zM13.009 29.585v-4.158h2.784v.786h-1.848v.87h1.644v.786h-1.644v1.716h-.936zM17.39 28.823l.15.012h.21c.468 0 .814-.118 1.038-.354.228-.236.342-.562.342-.978 0-.436-.108-.766-.324-.99-.216-.224-.558-.336-1.026-.336-.064 0-.13.002-.198.006-.068 0-.132.004-.192.012v2.628zm2.706-1.32c0 .36-.056.674-.168.942a1.73 1.73 0 01-.48.666 2.005 2.005 0 01-.75.396 3.491 3.491 0 01-.996.132c-.168 0-.364-.008-.588-.024a4.51 4.51 0 01-.66-.084v-4.05c.216-.04.44-.066.672-.078.236-.016.438-.024.606-.024.356 0 .678.04.966.12.292.08.542.206.75.378.208.172.368.392.48.66.112.268.168.59.168.966zM21.724 28.943c.164 0 .28-.016.348-.048.068-.032.102-.094.102-.186 0-.072-.044-.134-.132-.186a2.31 2.31 0 00-.402-.186 4.445 4.445 0 01-.384-.162 1.182 1.182 0 01-.288-.198.845.845 0 01-.186-.282 1.033 1.033 0 01-.066-.39c0-.296.11-.53.33-.702.22-.172.522-.258.906-.258.192 0 .376.018.552.054.176.032.316.068.42.108l-.156.696a3.207 3.207 0 00-.342-.096 1.802 1.802 0 00-.408-.042c-.28 0-.42.078-.42.234 0 .036.006.068.018.096a.225.225 0 00.072.084c.036.024.084.052.144.084.064.028.144.06.24.096.196.072.358.144.486.216.128.068.228.144.3.228.076.08.128.17.156.27.032.1.048.216.048.348 0 .312-.118.548-.354.708-.232.16-.562.24-.99.24-.28 0-.514-.024-.702-.072a2.85 2.85 0 01-.384-.12l.15-.726a2.413 2.413 0 00.942.192z" fill="#fff"/></svg>
```

Comment:
Same concern, can we have one of these new logos and reuse it in other places?

Reply:
These are the icons that we had before rebranding. Should I remove them ?
Since some of our previous icons are kept in the components I was not sure if I should remove them or not.